hackathon_site/dashboard/frontend/src/slices/order/teamOrderSlice.test.ts

```diff
@@ -0,0 +1,165 @@
+import store, { makeStore, RootState } from "slices/store";
+import {
+    mockCheckedOutOrdersInTable,
+    mockOrders,
+    mockPendingOrdersInTable,
+    mockReturnedOrdersInTable,
+    mockTeam,
+} from "testing/mockData";
+import { makeMockApiListResponse, makeStoreWithEntities, waitFor } from "testing/utils";
+import { get } from "api/api";
+import { displaySnackbar } from "slices/ui/uiSlice";
+import thunk, { ThunkDispatch } from "redux-thunk";
+import { AnyAction } from "redux";
+import configureStore from "redux-mock-store";
+import {
+    teamOrderReducerName,
+    initialState,
+    teamOrderSliceSelector,
+    isLoadingSelector,
+    errorSelector,
+    teamOrderSelectors,
+    hardwareInOrdersSelector,
+    getAdminTeamOrders,
+    returnedOrdersSelector,
+    pendingOrdersSelector,
+    checkedOutOrdersSelector,
+    TeamOrderState,
+} from "./teamOrderSlice";
+import exp from "constants";
+
+jest.mock("api/api", () => ({
+    ...jest.requireActual("api/api"),
+    get: jest.fn(),
+    patch: jest.fn(),
```

Comment:
The patch function hasn't been used in this test file, you should remove it if it's not being used